gqueries/output_elements/output_series/bezier_56_heat_production_in_households/solar_used_for_heating_in_households.gql

```diff
@@ -4,5 +4,6 @@
 - query =
     SUM(
       V(CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater)),input_of_solar_radiation),
-      V(CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater)),input_of_solar_thermal)      
+      V(CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater)),input_of_solar_thermal),
```

Comment:
None of the child nodes (input nodes) have a solar radiation or solar thermal carrier input, and so will always return zero. Is this intentional?

```ruby
CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater))
  .select { |conv| conv.input(:solar_thermal) }

# => [ <empty> ]

CHILDREN(V(households_useful_demand_for_space_heating_after_insulation_and_solar_heater))
  .select { |conv| conv.input(:solar_radiation) }

# => [ <empty> ]